ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -84,7 +85,7 @@ Verify RHODS Installation
   ${dashboard} =    Is Component Enabled    dashboard    ${DSC_NAME}
   IF    ("${UPDATE_CHANNEL}" == "stable" or "${UPDATE_CHANNEL}" == "beta") or "${dashboard}" == "true"
     # Needs to be removed ASAP
-    IF  "${UPDATE_CHANNEL}" == "odh-nightlies"
+    IF  "${PRODUCT}" == "ODH"
```

Comment:
Yeah, let's hope that this won't break our automation - did a brief look for our nightly and hopefully shouldn't :crossed_fingers:

Reply:
I had tested it for this code with RHODS and ODH operator by running below cmd :

```
./run_robot_test.sh --include install --test-variable TEST_ENV:CRC --test-variable cluster_type:selfmanaged --test-variable RHODS_OSD_INSTALL_REPO
:https://gitlab.cee.redhat.com/data-hub/olminstall.git --test-variable INSTALL_TYPE:OperatorHub --skip-oclogin true --test-variable UPDATE_CHANNEL:fast --test-case tasks/Tasks/rhods_olm.robot --test-variab
les-file test-variables.yml

```